src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -630,8 +630,7 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
    * According to stripe docs https://stripe.com/docs/js/element/events/on_change?type=paymentElement the change event has
```

Comment:
also I'd be happier if we didn't have to listen to changes in the payment element and instead this data came as part of the `Promise<PaymentIntentResult>` returned when we call `Stripe.confirmPayment`. But `PaymentIntentResult` just consists of the intent or an error.

Reply:
Right - done in 86713a52